driver/conv_driver.hpp

```diff
@@ -2955,7 +3124,11 @@ int ConvDriver<Tgpu, Tref>::VerifyForward()
         std::cout << "Forward Convolution Failed: " << error << " > " << tolerance << std::endl;
         return EC_VerifyFwd;
     }
-    std::cout << "Forward Convolution Verifies on CPU and GPU (" << error << ')' << std::endl;
+    if(miopen::IsEnabled(MIOPEN_DRIVER_USE_GPU_REFERENCE{}))
+        std::cout << "Forward Convolution Verifies on GPU(ref) and GPU (" << error << ')'
+                  << std::endl;
+    else
+        std::cout << "Forward Convolution Verifies on CPU and GPU (" << error << ')' << std::endl;
```

Comment:
Please revert this. You can indicate that driver uses GPU reference somewhere else in the driver's output.

Reply:
OK... let me find another place to print out

Reply:
Well, actually "Verifies on CPU and GPU" would be incorrect for the new case... Let me think about it.

Reply:
Sure.